addons/zeus/functions/fnc_moduleStaticWeaponPlace.sqf

```diff
@@ -0,0 +1,65 @@
+/*
+ * Author: PabstMirror
+ * Adapted from BIS_fnc_packStaticWeapon (Dean "Rocket" Hall, reworked by Killzone_Kid)
+ * Place on a unit in a group carrying static weapon backpacks to make them setup the weapon
+ *
+ * Arguments:
+ * 0: Module logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [LOGIC] call ace_zeus_fnc_moduleStaticWeaponPlace
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_logic"];
+if (!local _logic) exitWith {};
+
+if (canSuspend) exitWith {[FUNC(moduleStaticWeaponPlace), _this] call CBA_fnc_directCall;};
+
+// Validate the module target
+private _unit = effectiveCommander (attachedTo _logic);
+private _group = group _unit;
+TRACE_3("moduleStaticWeaponPlace placed",_unit,typeOf _unit,typeOf _logic);
+
+deleteVehicle _logic; // cleanup logic now, we just needed it to get the attached unit
+
+if (isNull _unit) exitWith { [LSTRING(NothingSelected)] call FUNC(showMessage); };
+if (!alive _unit) exitWith { [LSTRING(OnlyAlive)] call FUNC(showMessage); };
+if (!(_unit isKindOf "CAManBase")) then { [LSTRING(OnlyInfantry)] call FUNC(showMessage); };
+if (({[_x] call EFUNC(common,isPlayer)} count units _group) != 0) exitWith { ["str_a3_cfgvehicles_moduleremotecontrol_f_errorPlayer"] call FUNC(showMessage); };
+
+private _gunner = objNull;
+private _assistant = objNull;
+private _availableUnits = [_unit]; // try to seach attached unit first
+_availableUnits append ((units _group) select {(!isNull _x) && {_unit != _x}});
+_availableUnits = _availableUnits select {([_x] call EFUNC(common,isAwake)) && {(vehicle _x) == _x}};
+
+{
+    if ((unitBackpack _x) isKindOf "Weapon_Bag_Base") exitWith {_gunner = _x};
```

Comment:
This will trigger for UAV bags (how ever UAVs have a blank value for `assembleInfo >> base` so they will exit there, but with a confusing message. Maybe check if the assemble to string contains UAV?

Sub note: while not in vanilla crew served weapons, what happens when the `base` is an empty string (a single backpack setting up a turret)? Should that case be handled at all?